src/js/components/RangeSelector/index.d.ts

```diff
@@ -8,7 +8,7 @@ export interface RangeSelectorProps {
   max?: number;
   messages?: { lower?: string; upper?: string };
   min?: number;
-  onChange?: (...args: any[]) => void;
+  onChange?: (event: React.ChangeEvent) => void;
```

Comment:
Should this be `React.ChangeEvent<HTMLInputElement>`?
```suggestion
  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
```